[2025-06-22T15:47:03.281+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-22T15:47:03.302+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: start_kafka_hadoop_and_run_notebooks.start_zookeeper manual__2025-06-22T08:46:35.834149+00:00 [queued]>
[2025-06-22T15:47:03.307+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: start_kafka_hadoop_and_run_notebooks.start_zookeeper manual__2025-06-22T08:46:35.834149+00:00 [queued]>
[2025-06-22T15:47:03.307+0700] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2025-06-22T15:47:03.321+0700] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): start_zookeeper> on 2025-06-22 08:46:35.834149+00:00
[2025-06-22T15:47:03.326+0700] {standard_task_runner.py:63} INFO - Started process 48733 to run task
[2025-06-22T15:47:03.330+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'start_kafka_hadoop_and_run_notebooks', 'start_zookeeper', 'manual__2025-06-22T08:46:35.834149+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/start_DAG.py', '--cfg-path', '/tmp/tmpxhyfixtv']
[2025-06-22T15:47:03.331+0700] {standard_task_runner.py:91} INFO - Job 9: Subtask start_zookeeper
[2025-06-22T15:47:03.366+0700] {task_command.py:426} INFO - Running <TaskInstance: start_kafka_hadoop_and_run_notebooks.start_zookeeper manual__2025-06-22T08:46:35.834149+00:00 [running]> on host Panda
[2025-06-22T15:47:03.429+0700] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='group_03' AIRFLOW_CTX_DAG_ID='start_kafka_hadoop_and_run_notebooks' AIRFLOW_CTX_TASK_ID='start_zookeeper' AIRFLOW_CTX_EXECUTION_DATE='2025-06-22T08:46:35.834149+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-22T08:46:35.834149+00:00'
[2025-06-22T15:47:03.430+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-22T15:47:03.431+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-22T15:47:03.431+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'nohup /home/panda/kafka_2.13-3.9.1/bin/zookeeper-server-start.sh /home/panda/kafka_2.13-3.9.1/config/zookeeper.properties > /tmp/zookeeper.log 2>&1 &']
[2025-06-22T15:47:03.444+0700] {subprocess.py:86} INFO - Output:
[2025-06-22T15:47:03.447+0700] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-22T15:47:03.447+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-22T15:47:03.468+0700] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=start_kafka_hadoop_and_run_notebooks, task_id=start_zookeeper, execution_date=20250622T084635, start_date=20250622T084703, end_date=20250622T084703
[2025-06-22T15:47:03.500+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-22T15:47:03.528+0700] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-22T15:47:03.542+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
